README.md

```diff
@@ -13,8 +13,8 @@ function calls validation, script imports verification...
 
 `bslint` is:
 
-- a CLI tool to lint your code without compiling your project,
-- a `brighterscript` plugin offering **additional insights** on your code.
+-   a CLI tool to lint your code without compiling your project,
```

Comment:
It appears that this entire file somehow got formatted. Can you undo that change, and only add the new changes you directly modified? 

Also, can you add bullet points in the `### Code style rules` region for these two new rules. That's a great place to explain what the rules are, and why developers should consider enabling them.

Reply:
Readme is adjusted